package.json

```diff
@@ -59,13 +57,16 @@
     "node-fetch": "^3.3.2",
     "qrcode": "^1.5.3",
     "run-node-webpack-plugin": "^1.3.0",
-    "tailwindcss": "^3.4.15",
     "webpack": "5.96.1",
     "webpack-cli": "^5.1.4",
     "webpack-node-externals": "^3.0.0",
     "xml2js": "^0.6.2"
   },
   "devDependencies": {
+    "tailwindcss": "^4.1.6",
+    "@tailwindcss/cli": "^4.1.6",
+    "@tailwindcss/typography": "^0.5.16",
+    "daisyui": "^5.0.35",
```

Comment:
Moved these all to devDependencies since they aren't used in the JS code itself and they still function the same as dev dependencies.